pyproject.toml

```diff
@@ -11,3 +11,6 @@ requires = [
     "numpy>=1.26.0b1; python_version>='3.12'"
 ]
 build-backend = 'setuptools.build_meta'
+
+[tool.cibuildwheel]
```

Comment:
Do we still need this now that numpy 1.26 is out?

Reply:
Oops no will remove